annotate-equations.tex

```diff
@@ -242,7 +241,32 @@ \subsection{Default formatting of annotation labels}
 \vspace{1em}
 \end{LTXexample}
 \noindent
+
+\verb|\eqnannotationstrut| is defined to be a strut to provide minimum distance
+from text to the surrounding diagrams. By default it is \verb|\strut|, which has
+a similar effect to \verb|\mathstrut| in \verb|\eqnhighlightheight|.
+
+\begin{LTXexample}[text outside listing,lefthand width=0.5in]
+% Default
+\renewcommand{\eqnannotationstrut}{\strut}
+\begin{equation*}
+	\eqnmarkbox[blue]{v}{v}
+\end{equation*}
+\annotate[yshift=-0.5em]{below}{v}{velocity}
```

Comment:
```suggestion
    \eqnmarkbox[blue]{size}{s}
\end{equation*}
\annotate[yshift=-0.5em]{below}{size}{the size}
```
I think the effect is more striking with text that doesn't have descenders (like in `y`).
also, changed the tab to spaces for consistency with the rest of the formatting.